manifests/capacityscheduling/eq1.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: capacityscheduling.sigs.k8s.io/v1alpha1
```

Comment:
```suggestion
apiVersion: scheduling.sigs.k8s.io/v1alpha1
```

And name this file as "elasticquota.yaml"

Reply:
Sure I rename it as `elasticquota-example.yaml`